src/Header/Header.css

```diff
@@ -0,0 +1,76 @@
+.page-header {
```

Comment:
Can try making navbar sticky by adding these styles to "page-header" class-

  position: sticky;
   top: 0;
   z-index: 1;
   background-color: white;